src/components/inputs/z-select/index.tsx

```diff
@@ -102,6 +105,11 @@ export class ZSelect {
     this.selectedItem = this.itemsList.find((item: SelectItem) => item.selected);
   }
 
+  @Listen("ariaDescendantFocus")
+  getFocusedItemHandler(e: CustomEvent): void {
+    this.focusedItemId = (e.target as Element).id;
```

Comment:
Questo valore andrebbe resettato all'apertura e/o alla chiusura della tendina, perché quando apri la tendina il focus è sempre in cima, quindi non è coerente